lib/linguist/languages.yml

```diff
@@ -947,6 +947,18 @@ F#:
   tm_scope: source.fsharp
   ace_mode: text
 
+F*:
+  type: programming
+  color: "#b845fc"
+  search_term: fstar
+  aliases:
+  - fstar
+  extensions:
+  - .fst
+  - .fsti
```

Comment:
I think you should remove that extension. There are [only 6 `.fsti` files](https://github.com/search?utf8=%E2%9C%93&q=NOT+djchfdjbhdjfbfnv+extension%3Afsti&type=Code&ref=searchresults) ;)